.github/workflows/test_native.yml

```
@@ -250,24 +260,102 @@ jobs:
         with:
-          submodules: recursive
+          persist-credentials: false
 
-      - name: Setup native build
+      - name: Setup native test build
         id: base
-        uses: ./.github/actions/setup-native
+        uses: ./.github/actions/setup-native-test
 
       - name: Get release version string
         run: echo "long=$(./bin/buildinfo.py long)" >> $GITHUB_OUTPUT
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:43: Double quote to prevent globbing and word splitting

(shellcheck)